package com.github.lalifeier.mall.cloud.common.exception;

public class InternalException extends RuntimeException {
}
